terraform-project-api-gateway_module/variables.tf

```diff
@@ -1,9 +1,16 @@
-variable "ssm_region" {
+variable "region" {
   type        = string
   description = "Region"
+  default     = "us-west-2"
```

Comment:
Is there a way to dynamically determine the region instead of hard-coding it here?  This makes this terraform not portable.